NEWS.md

```diff
@@ -1,7 +1,7 @@
 # ggplot2 (development version)
 
-* Added weight aesthetic option to `stat_density()` and made scaling of 
-  weights the default (@annennenne, #2902)
+* Changed `theme_grey()` setting for legend key so that it creates no 
```

Comment:
It looks you accidentally overrode this news bullet rather than adding a new one.

Reply:
Woops, reinserted it!